test/policy-collection/policy_gatekeeper_operator_test.go

```diff
@@ -238,7 +238,7 @@ var _ = Describe("", func() {
 				"--kubeconfig="+kubeconfigHub)
 			By("Patching to remove dryrun")
 			utils.KubectlWithOutput(
-				"patch", "-n", userNamespace, common.GvrPolicy.Resource+"."+common.GvrPlacementBinding.Group, GKPolicyName,
+				"patch", "-n", userNamespace, gvrPolicy.Resource+"."+gvrPlacementBinding.Group, GKPolicyName,
```

Comment:
Oops. Thanks for fixing this! I'm not sure how it's been working in the canaries before this, though!

...oh. Because it hasn't been running 🙁 :
```
Failed to compile policy-collection:
go: inconsistent vendoring in /go/src/github.com/open-cluster-management/governance-policy-framework:
	github.com/onsi/gomega@v1.17.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
	github.com/onsi/gomega@v1.16.0: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
	To ignore the vendor directory, use -mod=readonly or -mod=mod.
	To sync the vendor directory, run:
		go mod vendor
```
https://app.travis-ci.com/github/open-cluster-management/canary/jobs/552464012#L512-L520

Reply:
#231